autopush/router/fcm.py

```diff
@@ -32,8 +31,87 @@ def __init__(self, ap_settings, router_conf):
         self.auth = router_conf.get("auth")
         self.metrics = ap_settings.metrics
         self._base_tags = []
+        self.creds = {}
+        self.regid = ""
+        self.reasonTable = {
```

Comment:
I'd prefer reasonTable as a global var, or as a class var like APNSRouter.errors since it's a big chunk of immutable data